lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -548,6 +548,22 @@ void HdVP2BasisCurves::Sync(
         return;
     }
 
+    // We don't update the repr if it is hidden by the render tags (purpose)
+    // of the ProxyRenderDelegate. In additional, we need to hide any already
+    // existing render items because they should not be drawn.
+    auto* const          param = static_cast<HdVP2RenderParam*>(_delegate->GetRenderParam());
+    ProxyRenderDelegate& drawScene = param->GetDrawScene();
+    if (!drawScene.DrawRenderTag(delegate->GetRenderIndex().GetRenderTag(GetId()))) {
+        _HideAllDrawItems(reprToken);
+        *dirtyBits &= ~( HdChangeTracker::DirtyRenderTag
+#if USD_VERSION_NUM <= 2005
```

Comment:
Pixar is in the process of fixing this one, hopefully the fix is in the next version. If node we may have to adjust this check.

Reply:
People may have already adopted a newer version of USD and won't get proper behavior because of above assumption. Let's not assume things will get fixed...but instead document places which will require conditional compilation when it will be available in USD.

One way of doing it is to create a new macro which will say ENABLE_RENDERTAG_VISIBILITY_WORKAROUND and add a comment in there pointing to GitHub issue in USD (if we have one)...and/or usd-interest discussion.

Reply:
Yea that is a nice way of doing it. I can put the documentation in the code once where I define the macro, and then the rest of the places test the macro and don't have to say much, because the macro explains why that code is there already.